main/src/main/scala-3/com/typesafe/scalalogging/LoggerImpl.scala

```diff
@@ -54,4 +55,87 @@ trait LoggerImpl {
   inline def trace(inline marker: Marker, inline message: String, inline cause: Throwable): Unit = ${LoggerMacro.traceMessageCauseMarker('underlying, 'marker, 'message, 'cause)}
   inline def trace(inline marker: Marker, inline message: String, inline args: Any*): Unit = ${LoggerMacro.traceMessageArgsMarker('underlying, 'marker, 'message, 'args)}
   inline def whenTraceEnabled(inline body: Unit): Unit = ${LoggerMacro.traceCode('underlying, 'body)}
+
+  //scala 2
```

Comment:
the rest is a copy of the scala 2 macro definitions to be compiled in scala 3.  Then these can be used from either scala 3 or 2.